ecommerce/extensions/iap/processors/base_iap.py

```diff
@@ -212,6 +212,7 @@ def record_processor_response(self, response, transaction_id=None, basket=None,
                                                                             basket=basket)
 
         meta_data = self._get_metadata(currency_code=currency_code, price=price)
+        original_transaction_id = original_transaction_id or transaction_id
```

Comment:
question: how does this change relate to the inapp price change?

Reply:
It doesn't. I noticed that nit and will be sending as a separate commit in this PR.